src/components/pages/Map.tsx

```diff
@@ -92,6 +63,41 @@ function Map() {
   }, []);
   // </表示範囲判定>
 
+  // <infoWindowオプション-->
+  const [size, setSize] = useState<undefined | google.maps.Size>(undefined);
+  const infoWindowOptions = {
+    pixelOffset: size
+  };
+  const createOffsetSize = () => {
+    return setSize(new window.google.maps.Size(0, -45));
+  };
+
+  const [LatLng, setLatLng] = useState<google.maps.LatLng | google.maps.LatLngLiteral>();
+  // </infoWindowオプション-->
+
+  // <座標データ取得 未実装>
+  const [shops, setShops] = useState([]);
+  type shops = {
```

Comment:
useStateのgetter名と型名が重複しちゃってるので型名を変えましょう！
```suggestion
  const [shops, setShops] = useState([]);
  type Shop = {
```